client/src/main/java/org/eclipse/hono/client/device/amqp/CommandResponder.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.client.device.amqp;
+
+import java.util.Map;
+
+import org.eclipse.hono.client.ServiceInvocationException;
+import org.eclipse.hono.client.device.amqp.internal.AmqpSenderLink;
+
+import io.vertx.core.Future;
+import io.vertx.proton.ProtonDelivery;
+
+/**
+ * A sender to send back the response message of a command to Hono's AMQP adapter.
+ */
+public interface CommandResponder extends AmqpSenderLink {
```

Comment:
IMHO, `CommandResponseSender` would make it more obvious at first glance, that there is a `sender` link behind this class.

Reply:
Yes, that is more obvious. This already exists in the client package. That's why I hesitated to use the same name because it might be too difficult to distinguish between them just by the package.

Reply:
Ok, I see.